src/z3c/wizard/zcml.py

```diff
@@ -110,20 +99,18 @@ class IWizardStepDirective(zope.interface.Interface):
         Multiple interfaces can be provided, separated by
         whitespace.""",
         required=False,
-        value_type=zope.configuration.fields.GlobalInterface(),
-        )
+        value_type=zope.configuration.fields.GlobalInterface())
 
     allowed_attributes = zope.configuration.fields.Tokens(
-        title=u"View attributes that are also allowed if the user"
-                " has permission.",
+        title=u"View attributes that are also allowed if the user "
+              u"has permission.",
```

Comment:
Personally I prefer the blank to be on the 2nd line -- makes it easier to visually check that it's not missing.